### ISSUE TITLE
Java Autoformat keeps adding a new line to source files.

### ISSUE THREAD

**Issue by tylervangorder-8451** [openrewrite/rewrite#3191]
Running `org.openrewrite.java.format.AutoFormat` with the maven plugin 4.45.0, we are seeing a new line added before the first annotated enumeration: 

Orignal Source:

```
import com.fasterxml.jackson.annotation.JsonProperty;

public enum Status {
  
  @JsonProperty("not-started")
  NOT_STARTED,
  @JsonProperty("started")
  STARTED
}
```

After first run: 

```
import com.fasterxml.jackson.annotation.JsonProperty;

public enum Status {

  
  @JsonProperty("not-started")
  NOT_STARTED,
  @JsonProperty("started")
  STARTED
}
```

After second run: 

```
import com.fasterxml.jackson.annotation.JsonProperty;

public enum Status {


  
  @JsonProperty("not-started")
  NOT_STARTED,
  @JsonProperty("started")
  STARTED
}
```

**Comment by knutwannheden** [comment]
Probably related to the fix of #3089 .

**Comment by tylervangorder-8451** [comment]
🎉 Thank you!
